_stbt/core.py

```diff
@@ -585,6 +585,23 @@ def load_image(filename, flags=cv2.IMREAD_COLOR):
     return image
 
 
+def crop(frame, region):
+    """Returns a new image that is a copy of the specified region of ``frame``.
+
+    :type frame: `stbt.Frame` or `numpy.ndarray`
+    :param frame: An image in OpenCV format (for example as returned by
+      `frames`, `get_frame` and `load_image`, or the ``frame`` parameter of
+      `MatchResult`).
+
+    :type Region region: The region to crop.
+
+    :returns: An OpenCV image (`numpy.ndarray`) containing the specified region
+      of the source frame. This is a copy, so you can change the data in the
+      image without changing the original video frame.
+    """
+    return _crop(frame, region).copy()
+
+
 def _crop(frame, region):
     if not _image_region(frame).contains(region):
         raise ValueError("'frame' doesn't contain 'region'")
```

Comment:
Maybe spruce up this exception to give the dimensions of the frame and the region

Reply:
Done.